.github/workflows/main.yml

```diff
@@ -173,6 +173,16 @@ jobs:
     secrets:
       NPM_TOKEN: ${{ secrets.NPMJS_PAT }}
 
+  publish-tezos-client-plugin-to-npm:
```

Comment:
@GImbrailo , it's this workflow here that's in the SRE wheelhouse and needing approval